src/components/views/Metric.js

```diff
@@ -14,17 +15,23 @@ export default withRouter(props => {
 
     function buildNValuesDL(nValues) {
         let pairs = [];
+        let i = 0;
```

Comment:
There's a neat little trick that can make this more concise. Instead of this, we could replace the `for ... in` below with this:

```js
Object.keys(nValues).forEach((populationName, i) => {
    // ...
});
```

I'll let you decide which style you prefer. :smiley:

Reply:
only difference with my approach is we can increment the `i` when `hasOwnProperty` checks out though it might not be necessary depending on `nValues`

Reply:
`Object.keys` will only return the manually-added properties, but I'll leave it up to you. It's just a stylistic thing. :smiley: